### ISSUE TITLE
Back-tick merges with aphabet just before it (ligatures turned off)

### ISSUE THREAD

**Issue by lokeshlkr** [arrowtype/recursive#463]
**Problem**
```

``         -- Normal Back-ticks 
`a` `b` `c`   -- Back-tick with alphabet Visually takes space for 1 character.
             While typing seems i have a space but actually its not there.
             please allow any option to disable it, i have ligatures turned off. 
```

This is very annoying when writing markdown files.

**Expected behavior**
Back-ticks to maintain their character position/visual. OR maybe the merged hybrid character can be modified to take up space for both characters.

**Screenshots**
![image](https://user-images.githubusercontent.com/23016050/117993303-3bb54d00-b35d-11eb-8024-206088e05de0.png)

**To Reproduce**
1. Use "Recursive Mono Casual Static" in any editor
2. Write anything in back-ticks `` such that there is an alphabet just before last back-tick.

**Environment (please complete the following information):**
- OS: Issue seems to be not related to OS, but tested and present in Windows and Linux.

**Comment by arrowtype** [comment]
Hi @lokeshlkr, thanks for taking the time to submit this issue!

Have you tried this with [Release 1.078](https://github.com/arrowtype/recursive/releases/tag/v1.078)? I think the issue was solved there, but please let me know if you are still facing the issue after installing the updated fonts.

**Comment by lokeshlkr** [comment]
Hi @arrowtype , 
Thanks for quick reply.
The issue is fixed in 1.078, i was using 1.077.
Sorry for my ignorance, i just looked at the open issues, should have seen closed ones.
And also a huge thank you for this awesome font. 
-keep smiling-

**Comment by arrowtype** [comment]
Awesome! No worries at all; I’m just glad things are working for you now. Sorry about the inconvenience of that temporary issue!

Thanks for the kind words. Be well!
